task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -935,6 +937,7 @@ def supervise(
     Run a single task execution to completion.
 
     :param ti: The task instance to run.
+    :param dr: Current DagRun of the task instance.
```

Comment:
This does not exist in the actual args? (actually `dag_path` below also does not exist; it’s `dag_rel_path` in the args)

Reply:
Fixed. Leftover from going few iterations.